libs/blocks/library-config/library-config.js

```diff
@@ -32,27 +32,47 @@ async function loadPersonalization(content, list) {
   personalization(content, list);
 }
 
-function addSearch(content, list) {
+function addSearch(content, list, type) {
```

Comment:
Same as above. I think it would be good to pass an object.
```suggestion
function addSearch({ content, list, type }) {
```

Reply:
Done